wandb/sdk/internal/internal_api.py

```diff
@@ -43,7 +43,9 @@
 from wandb.errors import AuthenticationError, CommError, UnsupportedError, UsageError
 from wandb.integration.sagemaker import parse_sm_secrets
 from wandb.old.settings import Settings
+from wandb.proto.wandb_internal_pb2 import ServerFeature
 from wandb.sdk.artifacts._validators import is_artifact_registry_project
+from wandb.sdk.internal._generated import SERVER_FEATURES_QUERY_GQL, ServerFeaturesQuery
```

Comment:
this might be a problematic change, since we want to support pydantic <2 and currently this generated folder is not compatible with pydantic<2. 

cc: @tonyyli-wandb

Reply:
yep, i left the additional context in the thread.

Reply:
resolved in slack thread